app/views/assessment/endScreens/Caregiver.js

```diff
@@ -1,32 +1,37 @@
 import React, { useContext } from 'react';
-import { Trans, useTranslation } from 'react-i18next';
+import { useTranslation } from 'react-i18next';
 
 import { Icons, Images } from '../../../assets';
-import { Typography } from '../../../components/Typography';
 import { MetaContext } from '../Context';
 import { Info } from '../Info';
+import { InfoText } from '../components/InfoText';
+import { Button } from '../components/Button';
 
 import { Colors } from '../../../styles';
 
 /** @type {React.FunctionComponent<{}>} */
 export const Caregiver = ({ navigation }) => {
-  let { t } = useTranslation();
-  let { completeRoute } = useContext(MetaContext);
+  const { t } = useTranslation();
+  const { completeRoute } = useContext(MetaContext);
```

Comment:
This MetaContext is pretty undescriptive it would be great to replace with something more focused in a future commit.

Reply:
Agreed, naming on this one is no good.